### DISCUSSION TITLE
Logbook name on logbook page

### DISCUSSION THREAD

**Discussion by jothebulb** [magicbug/Cloudlog#1468] [Ideas / Feature Requests]
Hi,

I was unable to find this request elsewhere, so apologies if this is a duplicate.

I use different logbooks, via the **Station Location** function, to keep separate logs for DMR contacts versus HF/VHF/UHF QSOs.
Currently the only way I  identify which location is in use is via the Account pulldown menu & checking which is the active logbook.. 

Is it possible to add a name field to the Logbook page that identifies the current location/logbook in use? It would be great if it was next to the 'Logbook' title at the top of the screen, so it would be possible to check at a glance which location/log was active.

**Comment by phl0** [comment]
Like that?

![Screenshot from 2022-05-02 12-33-50](https://user-images.githubusercontent.com/7112907/166221198-3d72ff27-4b84-4657-a64f-8a5c7825fcd5.png)

**Reply by jothebulb** [reply]
Just like that! 

Is that in the set-up somewhere? I am really sorry if I missed it.

Jonathan
G5LUX

**Reply by phl0** [reply]
Nope. I just added a bunch of lines of code to implement that.

https://github.com/phl0/Cloudlog/commit/798d6315fdd4fdd9283afc83433c938d01f01cf2

As for now just in my own repo atm. May require some more design before getting it added.

**Reply by jothebulb** [reply]
Fab - thank you for looking at this. I think it would be a useful detail to have.

73
Jonathan
G5LUX

**Reply by phl0** [reply]
Let's see what @magicbug has to say :)

**Reply by DF4OR** [reply]
+1 +1 +1 +1 +1 :)

73,
Ekki

**Reply by phl0** [reply]
This has been integrated into the station_logbooks branch with https://github.com/magicbug/Cloudlog/pull/1488.
